classes/mail/Mailable.inc.php

```diff
@@ -64,6 +65,13 @@ class Mailable extends IlluminateMailable
     public const GROUP_COPYEDITING = 'copyediting';
     public const GROUP_PRODUCTION = 'production';
 
+    /**
+     * The email variables handled by this mailable
+     *
+     * @param array<Variable>
+     */
+    public array $variables = [];
```

Comment:
public properties of `Mailable` class ancestors appear in the `Mailable::viewData` and then are passed to the compilation of the message's body. I haven't overridden this Laravel's behaviour. See `Illuminate\Mail\Mailable::buildViewData()`. This may lead to unexpected coercion. I would make it protected (together with `supportsTemplates` that I've missed) or remove it from data before compilation